src/prefect/experimental/workers/process.py

```diff
@@ -81,7 +103,7 @@ async def run(
     ):
         command = configuration.command
         if not command:
-            command = [sys.executable, "-m", "prefect.engine"]
+            command = (" ").join([sys.executable, "-m", "prefect.engine"])
```

Comment:
```suggestion
            command = f"{sys.executable} -m prefect.engine"
```
More of a nit than anything. I figure it makes it clearer that command is expected to be string, rather than a list that is subsequently joined. My feelings are especially strong about this, but I had a brief moment of wondering why we were joining a list just to split a list before realizing that it was to handle the non-default commands, which will all be strings.

Reply:
Oh yeah, that makes way more sense!